### DISCUSSION TITLE
I made a Libadwaita-like UI theme for the Godot game engine. It's themeable via Gradience

### DISCUSSION THREAD

**Discussion by ghost** [GradienceTeam/Gradience#643] [Show and tell]
[Godwaita](https://gitlab.com/zehkira/godwaita) is a UI theme for apps and games made in the Godot game engine. It's designed to look like Gtk+Libadwaita. It automatically adapts to the colors chosen via Gradience (by reading the `gtk-3.0/gtk.css` file).

Here you can see a window with the standard Godot theme on the left, and Godwaita on the right.

![](https://preview.redd.it/szlejqx5qyw91.png?width=1024&auto=webp&s=2f0d444d77902a996a5e730bc64544af935cb2f9)

**Comment by 0xMRTT** [comment]
Nice 👍
